src/test/ts/util.ts

```diff
@@ -91,14 +91,14 @@ describe('util', () => {
         ],
         [{ exclude: [] }, ['exclude'], []],
         [
-          { exclude: ['@scope/package'] }, // eslint-disable-line sonarjs/no-duplicate-string
```

Comment:
IMO `@scope/package` is fine. Let's add `test/ts/util.ts` to the rule exclusions instead.
https://github.com/antongolub/yarn-audit-fix/blob/master/src/test/lint/.eslintrc.json#L6

Reply:
Done: https://github.com/antongolub/yarn-audit-fix/pull/346/commits/13c91de7b3df15b2b92c141e88295324dce47c32